packages/e2e/spec/report-api.spec.ts

```diff
@@ -33,7 +33,7 @@ test.describe('Report api', () => {
     test('should accept v2 reports', async ({ baseURL }) => {
       const response = await client.uploadReport(
         simpleReportv2(
-          'github.com/stryker-mutator-test-organization/hello-org',
+          'github.com/stryker-mutator-test-organization/hello-org?realtime=true',
```

Comment:
I think this causes the e2e test to fail. Please create a separate e2e test for realtime.